nemo/collections/asr/models/label_models.py

```diff
@@ -546,21 +528,20 @@ def batch_inference(self, manifest_filepath, batch_size=32, sample_rate=16000, d
             embs: embeddings of files provided in manifest file
             logits: logits of final layer of EncDecSpeakerLabel Model
             gt_labels: labels from manifest file (needed for speaker enrollment and testing)
-            mapped_labels: Classification labels sorted in the order that they are mapped by the trained model
+            trained_labels: Classification labels sorted in the order that they are mapped by the trained model
```

Comment:
how about the groundtruth label does not belong to trained_label?  What whould be gt_labels in that case? 
If gt_label does not present and the model hasn't seen the class the audio, it will output most likely pred?

Reply:
gt_lables doesn;t depend on trained labels now, they just return what it gets from manifest.

Reply:
then could you add a test to ensure these conditions, please?

Reply:
No, we didn't have test for groundtruth label does not belong to trained_label. Could you add this?